Makefile

```diff
@@ -122,17 +128,17 @@ define generate_k8s_protobuf
 	    --go-header-file "$(PWD)/hack/custom-boilerplate.go.txt"
 endef
 
-build: $(SUBDIRS)
+build: $(SUBDIRS) ## Builds all the components for cilium by executing make in the respective sub directories.
```

Comment:
A few instances where the capitalization of "cilium" is inconsistent

Reply:
Thanks for pointing this out, I have updated the instances.